src/client/src/components/Comment.vue

```diff
@@ -29,9 +29,12 @@
 <template>
   <v-card color="rgb(211, 211, 211, 0)" class="my-n0" flat>
     <div v-if="!showEditForm">
-        <p class="author">{{author}} </p>
-        <p class="date">{{date}}</p><br/>
-        <p class="content">{{newContent}}</p><br/>
+      <v-list-item-avatar>
```

Comment:
The avatar is displacing the username but the rest of the card is still flushed left, the content and buttons should all line up with the Username. Check the figma mockup for reference.

Reply:
working on it rn